Build/Scripts/BuildCommon.js

```diff
@@ -207,5 +213,61 @@ namespace('build', function() {
 
     });
 
+ 
+  task('gendocs', {
+    async: true
+    }, function() {
+
+    fs.copySync(atomicRoot + "Build/Docs/Readme.md", jsDocFolder + "Readme.md");
+    fs.copySync(atomicRoot + "Build/Docs/atomic-theme", jsDocFolder + "atomic-theme");
+
+    cmds = [
+      "cd " + jsDocFolder + " && echo {} > package.json", // newer versions of npm require package.json to be in the folder or else it searches up the heirarchy
+      "cd " + jsDocFolder + " && npm install typedoc",
```

Comment:
We don't have a local npm so would be adding a dependency to build, not sure on typedoc's size, maybe it can go in: https://github.com/AtomicGameEngine/AtomicGameEngine/tree/master/Build/node_modules with other node build modules?